packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import { localize } from '@deriv/translations';
 
+type Category = { value: string; text: string };
+type TContract_type = {
+    name: string;
+    categories: Category[];
+};
+type Tcontract_types_list = {
+    [key: string]: TContract_type;
+};
+type TItem = {
+    value: string;
+};
+type TList = {
+    component?: null | React.ReactNode;
+    contract_types: Category[];
```

Comment:
no TContractType?

Reply:
no I mean. you defined TContractType above. is it not array of TContractTypes instead of array of Categories?